tutorials/2d/2d_sprite_animation.rst

```diff
@@ -79,11 +79,37 @@ released.
 
     extends KinematicBody2D
 
+    onready var _animated_sprite = $AnimatedSprite
```

Comment:
I don't see this as an improvement for such simple code. The documentation aims for simplicity and clarity, using fake private variables for everything goes against those objectives.

Reply:
Sure, it'll break apart cs/gds a bit but that can be changed.

Reply:
On the other hand, it's more performant to cache the node reference. I think it's a good idea to showcase good practices in the documentation.